objects/app/apps.go

```diff
@@ -154,21 +129,21 @@ func CreateApp(a *fnclient.Fn, app *modelsv2.App) (*modelsv2.App, error) {
 		return nil, err
 	}
 
-	fmt.Println("Successfully created app: ", resp.Payload.Name)
-	return resp.Payload, nil
+	fmt.Println("Successfully created app: ", resp.Name)
```

Comment:
It is probably worth getting the logging story worked out up front. I am new to this CLI code but maybe we could do better? Has this been considered?

Reply:
This is a great point! However, let's keep in mind that this effort is (narrowly) focussed on enabling the CLI to consume OCI APIs instead of OSS APIs. Keeping in mind time constraints it may be better to track the logging improvements in a separate issue.